app.py

```diff
@@ -144,6 +155,22 @@ def execute():
         'success': success
     })
 
+def get_conn_through_sentinel():
+    master_info = get_master(app.config['REDIS_URL'])
+    master_ip = str(master_info[0])
+    master_port = str(master_info[1])
+
+    conn = None
+    if app.config['SSL_ENABLED']:
+        ssl_cert_reqs = None if app.config['INSECURE_SSL'] else 'required'
+        conn = redis.Redis(host=master_ip, ssl=True,
+                           ssl_cert_reqs=ssl_cert_reqs,
+                           port=master_port,
+                           password=app.config['REDIS_PASSWORD'])
+    else:
+        conn = redis.Redis(host=master_ip, port=master_port, password=app.config['REDIS_PASSWORD'])
```

Comment:
maybe use a kwargs and avoid the duplication?

Reply:
ok